lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -51,8 +53,25 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         end
       end
 
+      def adapter
+        :yaml
+      end
+
       def extend_data(invocation)
-        JobInvocation.extend_data(invocation)
+        if option_show_inputs?
+          invocation['template_invocations']&.each do |template|
+            input_values = template['template_invocation_input_values']
+            values_for_host = {}
+            input_values&.each do |input_value|
+              values_for_host[input_value['template_input_name']] = input_value['value']
+            end
+            hosts = invocation.dig('targeting', 'hosts')
+            host_index = hosts&.index { |h| h['id'] == template['host_id'] } || template['host_id']
+            invocation['targeting']['hosts'][host_index].merge!({ :inputs => values_for_host })
```

Comment:
On previous line you assume `invocation['targeting']['hosts']` may be `nil`, but here you don't.

Also direct assignment instead of doing an in-place merge with a hash containing single key-value pair would probably be faster.

Reply:
Can't there be weird cases with 2 `invocation['template_invocations']` with the same `host_id`?

Reply:
No, that should not happen